src/User/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,106 @@
+import { CloseOutlined, ShoppingFilled } from "@ant-design/icons";
+import { Button, Drawer, InputNumber } from "antd";
+import React, { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  changeQuantityCart,
+  removeFromCart,
+} from "../../../redux/actions/cart.action";
+import "./Cart.scss";
+
+function Cart() {
+  const dispatch = useDispatch();
+  const { totalItems, cart } = useSelector((state) => state.cart);
+
+  const [visible, setVisible] = useState(false);
+
+  const changeQuantityProduct = (value, cart) => {
+    dispatch(changeQuantityCart({ id: cart.id, quantity: value }));
+  };
+
+  const removeProduct = (product) => {
+    dispatch(removeFromCart(product));
+  };
+
+  const showItemCart = (carts) => {
+    return carts.map((cart, index) => {
+      return (
+        <div className='cart-item' key={index}>
+          <div className='item-quantity'>
+            <InputNumber
+              size='small'
+              min={1}
+              max={100000}
+              defaultValue={cart.quantity}
+              onChange={(e) => {
+                changeQuantityProduct(e, cart);
+              }}
+            />
+          </div>
+
+          <div className='cart-image'>
+            <img src={cart.image} alt='cart images' />
+          </div>
+          <div className='cart-infor'>
+            <p className='infor-name'>{cart.name}</p>
+            <p className='infor-price'>${cart.price}</p>
+            <span className='infor-quantity'>
+              {cart.quantity} x {cart.weight}lb
+            </span>
+          </div>
+
+          <div className='cart-total'>
+            <p className='total-price'>
+              ${(cart.quantity * cart.price).toFixed(2)}
+            </p>
+          </div>
+          <div className='cart-remove'>
+            <CloseOutlined onClick={() => removeProduct(cart)} />
+          </div>
+        </div>
+      );
+    });
+  };
+
+  const totalPrice = cart.reduce((total, product) => {
+    total += product.price * product.quantity;
+    return total;
+  }, 0);
+
+  const showDrawer = () => {
+    setVisible(true);
+  };
+
+  const onClose = () => {
+    setVisible(false);
+  };
+
+  return (
+    <div className='cart'>
+      <div className='cart-button' onClick={showDrawer}>
```

Comment:
```suggestion
      <div className='cart-button' onClick={() => setVisible(true)}>
```
Có thể làm tương tự với event onClose của Drawer. Nếu chỉ action đơn giản như này thì không cần tạo func làm gì